plasma_framework/contracts/src/framework/ExitGameController.sol

```diff
@@ -70,7 +70,7 @@ contract ExitGameController is ExitGameRegistry {
      * @dev Accessible only from non quarantined exit games, uses a mutex
      */
     function deactivateNonReentrant() external onlyFromNonQuarantinedExitGame() {
-        require(mutex, "Not locked");
+        assert(mutex);
```

Comment:
@thec00n I put up the discussion of `assert` because it is quite easy to come up with a test that can trigger this assert. We just need an invalid ExitGame that violates our protocol, our only protection against it is the quarantined period.

Reply:
`I put up the discussion of assert because it is quite easy to come up with a test that can trigger this assert. We just need an invalid ExitGame that violates our protocol, our only protection against it is the quarantined period.`

If that is the only case where a tool can trigger an assert violation then that's fine. I suspect that they will not find this case because it's too complex create such a state for an automated tool. We are interested in assert violations apart from this scenario.